src/Codeception/Module/Symfony.php

```diff
@@ -12,80 +12,67 @@
 use Symfony\Component\VarDumper\Cloner\Data;
 
 /**
- * This module uses Symfony Crawler and HttpKernel to emulate requests and test response.
+ * This module makes it possible to access Symfony features. It can be loaded in two ways:
+ * * Complete: Requests are emulated through Symfony's [DomCrawler](http://symfony.com/doc/current/components/dom_crawler.html) and [HttpKernel](https://symfony.com/doc/2.7/components/http_kernel.html)
+ * * Certain parts only: [PhpBrowser Module](http://codeception.com/docs/modules/PhpBrowser) or [WebDriver Module](http://codeception.com/docs/modules/WebDriver) is used for requests
  *
- * ## Demo Project
- *
- * <https://github.com/Codeception/symfony-demo>
+ * ## Complete
  *
- * ## Status
+ * ### Config
  *
- * * Maintainer: **raistlin**
- * * Stability: **stable**
- *
- * ## Config
- *
- * ### Symfony 2.x
+ * ```yml
+ * # functional.suite.yml
+ * modules:
+ *     enabled:
+ *         - Symfony:
+ *             app_path: 'app' # Path to AppKernel.php
+ *             var_path: 'app' # Required for Symfony 3 directory structure only
+ *             # Optional parameters:
+ *             environment: 'local_test'
+ *             debug: true
+ *             em_service: 'doctrine.orm.entity_manager' # Use a different than the standard EntityManager to pair with Doctrine Module
+ *             cache_router: 'false' # Enable router caching between tests in order to increase performance
+ *             rebootable_client: 'true' # Reboot kernel before each request
+ * ```
  *
- * * app_path: 'app' - specify custom path to your app dir, where bootstrap cache and kernel interface is located.
- * * environment: 'local' - environment used for load kernel
- * * debug: true - turn on/off debug mode
- * * em_service: 'doctrine.orm.entity_manager' - use the stated EntityManager to pair with Doctrine Module.
- * * cache_router: 'false' - enable router caching between tests in order to [increase performance](http://lakion.com/blog/how-did-we-speed-up-sylius-behat-suite-with-blackfire)
- * * rebootable_client: 'true' - reboot client's kernel before each request
+ * ### Compatibility
  *
- * ### Example (`functional.suite.yml`) - Symfony 2.x Directory Structure
+ * When loaded this way, this module is compatible with all other Codeception modules, except WebDriver and the modules for other PHP frameworks.
+ * 
+ * ## Certain Parts Only
+ * 
+ * ### Config
  *
- * ```
- *    modules:
- *        - Symfony:
- *            app_path: 'app/front'
- *            environment: 'local_test'
+ * ```yml
+ * # acceptance.suite.yml
+ * modules:
+ *     enabled:
+ *         - Symfony:
+ *             part: 'services' # 'orm' or 'services'
+ *         - Doctrine2:
+ *             depends: Symfony
+ *         - WebDriver:
+ *             url: http://www.example.com/
+ *             browser: chrome
  * ```
  *
- * ### Symfony 3.x Directory Structure
```

Comment:
I found only one difference to Symfony 2, and explained it in the codeblock too.